docs/source/core_services/service_agents/actuator/ActuatorScheduleResponse.rst

```diff
@@ -10,7 +10,7 @@ ActuatorAgent will respond on the topic
     #python
     {
         'type': <'NEW_SCHEDULE', 'CANCEL_SCHEDULE'>
-        'requesterID': <Agent ID from the request>,
+        'requesterID': <Agent VIP identity from the request>,
```

Comment:
If we remove the argument then every module that works with the AA will break.

Eventually we'll add a replacement RPC function (not now as the interface of the RPC function needs to change in other ways) and we'll mark this method deprecated.